kpatch-build/create-diff-object.c

```diff
@@ -2866,6 +2866,10 @@ int main(int argc, char *argv[])
 	if (!hint)
 		ERROR("FILE symbol not found in base. Stripped?\n");
 
+	hint = strdup(hint);
+	if (!hint)
+		ERROR("failed to copy file name.");
+
```

Comment:
This is _really_ minor, but if we move these lines up into the search for STT_FILE, there would only be a single assignment to hint, like `hint = strdup(sym->name);`